### PR TITLE
Merge packages/common

### PR DESCRIPTION
**Pull request by marxeille**

### CODE REVIEW

File: packages/background/src/keyring/__mocks__/keystores.ts [hunk 1]
```diff
@@ -69,75 +69,78 @@ export const mockKeyStoreScryptLedger: KeyStore = {
...
-      dklen: 32,
-      n: 131072,
-      r: 8,
-      p: 1,
-    },
-    mac: "d9b684a86d046bfc3bebc9ee2666f11aa35fa50dea7404d0a1bc3e9d31e9c849",
-  },
-  addresses: undefined,
-};
+export const mockKeyStorePbkdf2PrivateKey = null;
```
Comment (toandq2009) [inline packages/background/src/keyring/__mocks__/keystores.ts:72]: need remove comment

File: packages/background/src/keyring/keyring.ts [hunk 2]
```diff
@@ -1353,10 +1360,12 @@ export class KeyRing {
 
     const privKey = this.loadPrivKey(getCoinTypeByChainId(chainId));
     const privKeyHex = Buffer.from(privKey.toBytes()).toString("hex");
-    const decryptedData = PRE.decryptData(privKeyHex, message[0]);
-    return {
-      decryptedData,
-    };
+    //TODO: This is comment for security proxy-recrypt-js lib
```
Comment (toandq2009) [inline packages/background/src/keyring/keyring.ts:1363]: If not using this method available delete

File: packages/background/src/keyring/keyring.ts [hunk 3]
```diff
@@ -1353,10 +1360,12 @@ export class KeyRing {
...
+    // const decryptedData = PRE.decryptData(privKeyHex, message[0]);
+    // return {
+    //   decryptedData
```
Comment (toandq2009) [inline packages/background/src/keyring/keyring.ts:1366]: similar above

File: packages/background/src/keyring/keyring.ts [hunk 4]
```diff
@@ -1373,11 +1382,13 @@ export class KeyRing {
 
     const privKey = this.loadPrivKey(getCoinTypeByChainId(chainId));
     const privKeyHex = Buffer.from(privKey.toBytes()).toString("hex");
-    const rk = PRE.generateReEncrytionKey(privKeyHex, message[0]);
+    //TODO: This is comment for security proxy-recrypt-js lib
+    // const rk = PRE.generateReEncrytionKey(privKeyHex, message[0]);
```
Comment (toandq2009) [inline packages/background/src/keyring/keyring.ts:1386]: similar above

**Review by toandq2009 — APPROVED** [review]
lgtm

**Review by haunv3 — APPROVED** [review]

**Pull request merged**